tests/microgrid/mock_api.py

```diff
@@ -89,6 +90,7 @@ def add_component(
         self,
         component_id: int,
         component_category: ComponentCategory.V,
+        max_current: Optional[int] = None,
```

Comment:
Also I wonder if `max_current` should be `Optional[Current]` type instead

Reply:
This gets changed to `Fuse` a few commit later, so this change is probably not worth the effort.

Reply:
Oh this is the `mock_api` class. Sorry, you are right, I misread the filename. Will change it to `Current`.

Reply:
Done